### DISCUSSION TITLE
Can I use Pyscript externally?

### DISCUSSION THREAD

**Discussion by ghost** [pyscript/pyscript#2088] [Q&A] [answered]
What I mean is that you can connect a css file to html by using <link rel="stylesheet" href="#"> but can you do the same with pyscript?

**Comment by WebReflection** [comment]
sorry .. what?

**Reply by ghost** [reply]
like, you can add external css as <link rel="stylesheet" href="#"> but can you do the same thing with pyscript. so <py-script href="#"></py-script>

**Reply by WebReflection** [reply, marked as answer]
you can create any script type py or mpy at runtime and it will work so I guess the answer is yes? I am still not fully sure I understand your question but I hope this helps

**Reply by ghost** [reply]
Thank you.
